incubator/schema-registry/values.yaml

```diff
@@ -86,14 +86,21 @@ sasl:
     #       name: "zookeeper-secret"
     #       key: "zokeeper-client-password"
 
+# Scheme-Registry Settings
+configurationOverrides:
+  # Needed to run with 1 Kafka Broker
+  offsets.topic.replication.factor: 1
+  # If you are running the schema-registry-ui, you may run into CORS issues;
+  # unset the following two lines to fix. 
+  # Source: https://github.com/Landoop/schema-registry-ui#prerequisites
+  # access.control.allow.methods: "GET,POST,PUT,OPTIONS"
+  # access.control.allow.origin: "*"
```

Comment:
Might be wise to try and convince people to put a specific hostname here rather than "*" for better security.